tests/fs-test.rs

```diff
@@ -11,13 +11,14 @@ fn new_file_test() {
 	let testfile = PathBuf::from("foo.txt");
 	if testfile.exists() {
 		println!("Removing existing file {}", testfile.display());
-		remove_file(&testfile).expect(&std::format!("Can't remove {}", testfile.display()));
+		remove_file(&testfile).unwrap_or_else(|_| panic!("Can't remove {}", testfile.display()));
 	}
 	let bin_path = build_hermit_bin("create_file");
 	run_simple_vm(bin_path);
 
 	assert!(testfile.exists());
-	let file_content = read("foo.txt").unwrap();
+	let file_content =
+		read("foo.txt").unwrap_or_else(|_| panic!("Can't remove {}", testfile.display()));
```

Comment:
Uhm, this is not where that error message should be.

Reply:
Fixed.